content-src/components/DiscoveryStreamComponents/DSCard/_DSCard.scss

```diff
@@ -56,6 +56,10 @@ $excerpt-line-height: 20;
     .source {
       font-size: 13px;
       color: $teal-80;
+      width: $domain-width;
```

Comment:
@Mardak trying to find solution for getting rid of hard coded width here, any inputs on how to inherit width from parent container, width: auto doesn’t seem to help. Thanks